config/defaults.js

```diff
@@ -41,6 +41,13 @@ module.exports = {
     randomizationFactor: 0.25
   },
 
+  features: {
+    relengAPIProxy: {
+      image: 'quay.io/djmitche/relengapi-proxy:0.0.2',
+      token: process.env.RELENG_API_TOKEN
```

Comment:
This is only necessary right now until secrets land and we can stuff it in there.